one_fm/one_fm/custom/issue.json

```diff
@@ -239,9 +239,138 @@
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2022-03-24 10:49:35.649533",
+   "default": null,
+   "depends_on": "eval:doc.communication_medium == 'WhatsApp'",
+   "description": null,
+   "docstatus": 0,
+   "dt": "Issue",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "whatsapp_number",
+   "fieldtype": "Data",
+   "hidden": 0,
+   "hide_border": 0,
+   "hide_days": 0,
+   "hide_seconds": 0,
+   "idx": 6,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_preview": 0,
+   "in_standard_filter": 0,
+   "insert_after": "communication_medium",
+   "label": "WhatsApp Number",
+   "length": 0,
+   "mandatory_depends_on": "eval:doc.communication_medium == 'WhatsApp'",
+   "modified": "2022-03-24 10:49:35.649533",
+   "modified_by": "Administrator",
+   "name": "Issue-whatsapp_number",
+   "no_copy": 0,
+   "non_negative": 0,
+   "options": null,
+   "owner": "Administrator",
+   "parent": null,
+   "parentfield": null,
+   "parenttype": null,
+   "permlevel": 0,
+   "precision": "",
+   "print_hide": 0,
+   "print_hide_if_no_value": 0,
+   "print_width": null,
+   "read_only": 1,
+   "read_only_depends_on": null,
+   "report_hide": 0,
+   "reqd": 0,
+   "search_index": 0,
+   "translatable": 1,
+   "unique": 0,
+   "width": null
+  }
+ ],
+ "custom_perms": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "amend": 0,
+   "cancel": 0,
+   "create": 1,
+   "creation": "2013-02-01 10:36:25",
+   "delete": 1,
+   "docstatus": 0,
+   "email": 1,
+   "export": 0,
+   "idx": 1,
+   "if_owner": 0,
+   "import": 0,
+   "modified": "2021-08-18 08:59:46.738459",
+   "modified_by": "j.poil@armor-services.com",
+   "name": "736eb131a4",
+   "owner": "Administrator",
+   "parent": "Issue",
+   "parentfield": "permissions",
+   "parenttype": "DocType",
+   "permlevel": 0,
+   "print": 1,
+   "read": 1,
+   "report": 1,
+   "role": "Support Team",
+   "select": 0,
+   "set_user_permissions": 0,
+   "share": 1,
+   "submit": 0,
+   "write": 1
+  },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "amend": 0,
+   "cancel": 0,
+   "create": 1,
+   "creation": "2021-08-18 08:59:46.819796",
+   "delete": 0,
+   "docstatus": 0,
+   "email": 1,
+   "export": 1,
+   "idx": 0,
+   "if_owner": 0,
+   "import": 0,
+   "modified": "2021-08-18 08:59:46.819796",
+   "modified_by": "j.poil@armor-services.com",
+   "name": "347d4ad4b2",
+   "owner": "j.poil@armor-services.com",
+   "parent": "Issue",
+   "parentfield": "permissions",
+   "parenttype": "DocType",
+   "permlevel": 0,
+   "print": 1,
+   "read": 1,
+   "report": 0,
+   "role": "Employee",
+   "select": 0,
+   "set_user_permissions": 0,
+   "share": 0,
+   "submit": 0,
+   "write": 1
   }
  ],
- "custom_perms": [],
```

Comment:
This is what we maintain in perms

Reply:
@pjamsheer please review

Reply:
@mymi14s While export Customisation you can uncheck export permission
![Screenshot 2022-03-27 at 2 05 11 PM](https://user-images.githubusercontent.com/20554466/160273640-ddfe7f99-df9c-4bcc-8f0e-daef6bd4635b.png)